src/cfnlint/rules/parameters/AllowedValue.py

```diff
@@ -57,16 +59,16 @@ def check_value_ref(self, value, path, **kwargs):
                             for index, allowed_value in enumerate(parameter_values):
                                 if str(allowed_value) not in allowed_value_specs:
                                     param_path = ['Parameters', value, 'AllowedValues', index]
-                                    message = 'You must specify a valid allowed value for {0} ({1}).\nValid values are {2}'
+                                    message = 'You must specify a valid allowed value for {0} ({1}).\n{2}'
```

Comment:
The newline here doesn't translate to json dumps.  Wondering if we should drop this.

Reply:
Yeah I think `AllowedValuesDescription` seems unnecessary in general

Reply:
Removed the newline and `AllowedValuesDescription`